WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -63,10 +79,10 @@ local methods = {
             elseif(IsShiftKeyDown()) then
                 local editbox = GetCurrentKeyBoardFocus();
                 if(editbox) then
-				    if (not fullName) then
-				      local name, realm = UnitFullName("player")
-					  fullName = name.."-"..realm
-					end
+                    if (not fullName) then
```

Comment:
i can seperate it. removing it might be difficult since i guess this where tabs previously.